app/src/main/java/com/animal_villa/dto/Animal.kt

```diff
@@ -1,4 +1,4 @@
 package com.animal_villa.dto
 
-class Animal {
+public class Animal {
```

Comment:
The default visibility modifier in Kotlin is public.  Does specifying public again decrease technical debt in any way?